nms/packages/magmalte/app/components/ActionTable.js

```diff
@@ -30,7 +30,7 @@ import FilterList from '@material-ui/icons/FilterList';
 import FirstPage from '@material-ui/icons/FirstPage';
 import FormControl from '@material-ui/core/FormControl';
 import LastPage from '@material-ui/icons/LastPage';
-import MaterialTable from 'material-table';
+import MaterialTable from '@material-table/core';
```

Comment:
Unfortunately we have to drop the `strict-local` above

Reply:
thanks!